code/style.css

```diff
@@ -1,3 +1,89 @@
+* {
+  margin: 0; 
+  padding: 0;  
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: content-box;
+  -webkit-font-smoothing: antialiased; 
+  text-rendering: optimizeLegibility;
+}
+
 body {
-  background: #FFECE9;
+  background: rgb(250, 238, 231);
+  font-family: 'roboto', sans-serif;
+}
+
+.user-container img {
+  width: 250px;
+  display: grid;
+  grid-template-columns: 1, 1fr;
+  justify-items: center;
+  place-items: center;
+  margin: 15px;
+  border-style: inset;
+  border-width: 4px;
+  border-color:rgb(50, 82, 136);
+}
+
+.user-container a {
+  margin: 10px;
+  color:black;
+  text-decoration: none;
+  display: grid;
+  grid-template-columns: 1, 1fr;
+  justify-items: center;
+  place-items: center;
+  margin-bottom: 35px;
+}
+
+.repo-container {
+  display: grid;
+  grid-template-columns: 1, 1fr;
+  justify-items: center;
+  place-items: center;
+  grid-gap: 40px;
+  margin-bottom: 55px;
+}
+
+.repo-container a {
+text-decoration: none;  
+color:rgb(36, 161, 156);
+}
+
+.chart {
+max-width: 350px; 
+max-height: 350px; 
+}
+
+.chart-container {
+display: grid;
+place-items: center;
+margin-top: 25px;
+}
+
+@media (min-width: 668px) {
+.repo-container {  
+grid-template-columns: repeat(2, 1fr);
+justify-items: center;
+place-items: center;
+}
+}
+
+@media (min-width: 1024px) {
+.repo-container {
+ grid-template-columns: repeat(3, 1fr); 
+}  
+
+.user-container a:hover {
+  background-color: black;
+  color: rgb(217, 96, 152);
+} 
+.user-container img:hover {
+  border-color:rgb(217, 96, 152);  
+} 
+
+.repo-container a:hover {
+  background-color: black;
+  color: rgb(217, 96, 152);
+    }  
```

Comment:
I'm very impressed with how you used media queries to get such a good responsiveness! I've written a lot of unnecessary code in these and it just won't look good! Your solution with grid is amazing!